index.js

```diff
@@ -3983,6 +3983,26 @@
   S.splitOn =
   def('splitOn', {}, [$.String, $.String, $.Array($.String)], splitOn);
 
+  //# splitOnRegex :: RegExp -> String -> Array String
+  //.
+  //. Returns the substrings of its second argument separated by evaluated
+  //. matches of its first argument.
+  //.
+  //. See also [`splitOn`](#splitOn).
+  //.
+  //. ```javascript
+  //. > S.splitOnRegex(/\s+/, 'foo  bar baz   qux')
+  //. ['foo', 'bar', 'baz', 'qux']
+  //. ```
```

Comment:
I like to avoid examples which could better be written with some other Sanctuary function. In this case, [`words`][1] would be clearer. I'm struggling to think of a simple, vaguely realistic example which can't be written with `splitOn`.


[1]: https://sanctuary.js.org/#words